containers/clojure/README.md

```diff
@@ -0,0 +1,134 @@
+# Clojure
+
+## Summary
+
+*Develop Clojure applications. Includes the Calva and clj-kondo extensions.*
+
+| Metadata                    | Value                 |
+|-----------------------------|-----------------------|
+| *Categories*                | Core, Languages       |
```

Comment:
Similarly, this should reference Community rather than Core.

Reply:
The "(Community)" has been added to both files.